public class Test4 {
    public static void main(String[] args) {

        // 강제 예외 발생 ( 개발자가 고의적으로 예외 발생)
        throw new ArithmeticException();
    }
}
